grpc-proto/proto/query.proto

```diff
@@ -55,6 +55,15 @@ message Uuid {
   fixed64 lsb = 2;
 }
 
+message BigInteger {
```

Comment:
Could we name these `Varint` and `Decimal` to match the CQL protocol types?

Reply:
done